flink-docs/src/main/java/org/apache/flink/docs/configuration/ConfigOptionsDocGenerator.java

```diff
@@ -260,10 +262,26 @@ private static String toHtmlTable(final List<OptionWithMetaInfo> options) {
 	private static String toHtmlString(final OptionWithMetaInfo optionWithMetaInfo) {
 		ConfigOption<?> option = optionWithMetaInfo.option;
 		String defaultValue = stringifyDefault(optionWithMetaInfo);
+		Documentation.TableMeta tableMeta = optionWithMetaInfo.field.getAnnotation(Documentation.TableMeta.class);
+		StringBuilder sb = new StringBuilder();
```

Comment:
Use a meaningful local field name? For example: `tags`, `metaStr`

Reply:
Updated it.